src/test/java/org/folio/moduserstest/RestVerticleIT.java

```diff
@@ -1291,4 +1296,60 @@ public void test5UserName(TestContext context) {
     });
   }
 
+  private Future<Void> getPatronBlockConditions(TestContext context) {
+    log.info("Getting patron blocks conditions");
+
+    Future<JsonObject> future = getJson(context, "/patron-block-conditions");
+
+    return future.map(proxies -> {
+      JsonArray patronBlockConditions = proxies.getJsonArray("patronBlockConditions");
+      if (patronBlockConditions.size() != 6) {
+        fail("Expected 6 entries, found " + patronBlockConditions.size());
+      }
+      return null;
+    });
+  }
+
+  private Future<Void> getPatronBlockCondition(TestContext context, String id) {
+    log.info("Getting patron blocks condition");
+
+    Future<JsonObject> future = getJson(context, "/patron-block-conditions/"
+      + id);
+
+    return future.map(proxies -> {
+      String actualId = proxies.getString("id");
+      if (!id.equals(actualId)) {
+        fail("Expected object was not found, found: " + actualId);
+      }
+      return null;
+    });
+  }
+
+  private Future<Void> putPatronBlockCondition(TestContext context, String id) {
+    log.info("Trying to update patron block condition");
+
+    JsonObject jsonObject = new JsonObject()
+      .put("id", id)
+      .put("name", "Maximum number of items charged out");
+
+    Future<HttpResponse<Buffer>> future = put("/patron-block-conditions/"
+      + id, encode(jsonObject));
+
+    return future.map(response -> {
+      assertStatus(context, response, 204);
+      return null;
+    });
+  }
+
+  private Future<Void> deletePatronBlockCondition(TestContext context, String id) {
+    log.info("Deleting Patron Block Condition");
+
+    Future<HttpResponse<Buffer>> future = delete("/patron-block-conditions/"
+      + id);
+
+    return future.map(response -> {
+      assertStatus(context, response, 204);
+      return null;
+    });
+  }
```

Comment:
It's probably too late for this issue but I would like to mention that you may use rest assured for the unit tests. Rest assured is more easy to use and understand than RestITSupport: http://rest-assured.io/

Reply:
tests have been updated